React/Base/RCTJSStackFrame.m

```diff
@@ -100,7 +100,7 @@ + (instancetype)stackFrameWithLine:(NSString *)line
                                      file:file
                                lineNumber:[lineNumber integerValue]
                                    column:[column integerValue]
-                                 collapse:@NO];
+                                 collapse:@NO.integerValue];
```

Comment:
This seems a smell. Can the `collapse` var be made a `BOOL` instead, at least internally to this class? If not, then at least it seems better to just pass `0`.

Reply:
Agreed. Shouldn't be a huge change.

Reply:
I changed it to `BOOL`.